Makefile

```diff
@@ -0,0 +1,138 @@
+# -*- makefile -*-
+# lind-wasm-apps unified build (hardened, with lmbench build wrapper)
+
+SHELL := /bin/bash
+.ONESHELL:
+.SHELLFLAGS := -eu -o pipefail -c
+
+# -------- Paths ---------------------------------------------------------------
+LIND_WASM_ROOT ?= $(HOME)/lind-wasm
+BASE_SYSROOT   ?= $(LIND_WASM_ROOT)/src/glibc/sysroot
+
+APPS_ROOT      := $(CURDIR)
+APPS_BUILD     := $(APPS_ROOT)/build
+APPS_OVERLAY   := $(APPS_BUILD)/sysroot_overlay
```

Comment:
One divergence from the original design here, but a good compromise: we’ve got a single build/sysroot_overlay instead of per-app sysroot_overlay dirs (e.g. libtirpc/sysroot_overlay, lmbench/sysroot_overlay). Functionally fine, but it means overlays are global rather than per-app sandboxes.